fix issues on login screen

AdityaSrivast opened the pull request:
### Description
- [x] Fix toggle logic.
- [x] Show the information of unsuccessful login to the user.
- [x] Provide back to `/register` link, similar to _Already have an account? Sign in_ at `/register`

Fixes #153 

### Type of Change:
**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

### Screenshots: 

![login-gif1](https://user-images.githubusercontent.com/31389740/75853143-7d6e6d80-5e13-11ea-94e4-16c47844c38e.gif)


### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] Any dependent changes have been merged

AdityaSrivast commented:
@YashMeh @sumedhe please review

YashMeh commented on senz-web/frontend/src/_actions/auth.js:
```diff
@@ -54,6 +54,7 @@ export const LoginAction = ({ email, password }, history) => {
         type: AUTHENTICATION_ERROR,
         payload: "Invalid"
       });
+      alert("Email or password entered is incorrect");
```
Can we have a component giving this message instead of an alert ?

AdityaSrivast replied:
@YashMeh on it. Thanks for pointing out.

AdityaSrivast commented:
@YashMeh please review

Changes made:
- [x] Added component for notification
- [x] Renamed the class to `Login` instead of `Register` in `Login.jsx`

AdityaSrivast commented:
@YashMeh please review.
Changes made:
- [x] added component for notification
- [x] rename `Register` to `Login` in `Login.jsx`
- [x] modified test in login.test.js
